core/app/models/spree/log_entry.rb

```diff
@@ -2,10 +2,57 @@
 
 module Spree
   class LogEntry < Spree::Base
+    # Classes used in core that can be present in serialized details
+    #
+    # Users can add their own classes in
+    # `Spree::Config#log_entry_permitted_classes`.
+    #
+    # @see Spree::AppConfiguration#log_entry_permitted_classes
+    CORE_PERMITTED_CLASSES = [
+      ActiveMerchant::Billing::Response,
+      ActiveSupport::TimeWithZone,
+      Time,
+      ActiveSupport::TimeZone
+    ].freeze
+
+    # Raised when a disallowed class is tried to be loaded
+    class DisallowedClass < ArgumentError
```

Comment:
This is a nit, but maybe a good old `RuntimeError` is semantically better here? It doesn't seem like the exception is caused by a bad method argument.

Reply:
Oh, yeah, it makes total sense. Changed!